docs/client/basic/toc.js

```diff
@@ -48,7 +48,7 @@ var sections = [
       item("Defining templates in HTML", {id: "defining-templates"}),
       item("Template.<em>name</em>.helpers", {longname: "Template#helpers"}),
       item("Template.<em>name</em>.events", {longname: "Template#events"}),
-      item("Template.<em>name</em>.rendered", {longname: "Template#rendered"}),
+      item("Template.<em>name</em>.onRendered", {longname: "Template#onRendered"}),
```

Comment:
Literally everyone is relying on this answer: http://stackoverflow.com/questions/26976603/meteor-load-javascript-file-but-dont-execute-execute-on-demand

I can't believe you want to break this link